knimin/lib/ag_jira.py

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env python
+from jira import JIRA
+
+
+def jira_connect(server='https://jira.atlassian.com',
+                 user=None, password=None):
+    """Connect to JIRA server
+
+    Parameters
+    ----------
+    server: str, optional
```

Comment:
It's optional cause it has a default value, which is valid. BTW optional in this context means that the user doesn't have to pass it ...